packages/core/EventManager.ts

```diff
@@ -907,4 +920,68 @@ export default class EventManager {
       );
     }
   }
+
+  private async createAllCRMEvents(event: CalendarEvent) {
+    const createdEvents = [];
+    for (const credential of this.crmCredentials) {
+      const crm = new CrmManager(credential);
+
+      let success = true;
+      const createdEvent = await crm.createEvent(event).catch((error) => {
+        success = false;
+      });
+      createdEvents.push({
+        type: credential.type,
+        appName: credential.appId || "",
+        uid: createdEvent?.id || "",
```

Comment:
We are using `getUid` method  to get the UID in CalendarManager::createEvent, shouldn't we reuse that?